Include upgrader version in metrics and inventory (#35188)
gravitational/teleport: docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```code
$ tctl inventory ls --upgrader=none --older-than=v1.2.3
Server ID                            Hostname        Services Agent Version Upgrader Upgrader Version
------------------------------------ --------------- -------- ------------- -------- ----------------
1e6578b6-9530-448e-8013-d32641324abb old.example.com Node     v1.1.1        none     none
...
```

> The Teleport Upgrader will receive occasional updates. The `tctl inventory ls` command will also display the installed version of the Teleport Upgrader:

Reviewer: nit: this could be read to imply that the upgrader also upgrades itself, which it does not do.  I'd suggest omitting the phrase "will receive occasional updates".  If you want to reinforce the idea that the upgrader is independently versioned, maybe just say that explicitly.  Ex:

```suggestion
The `tctl inventory ls` command will also display the installed version of the Teleport Upgrader, which is versioned independently from teleport itself:
```
